docs/openstack/backend_services_deployment.md

```diff
@@ -197,11 +197,20 @@ podified OpenStack control plane services.
     ovn:
       enabled: false
       template:
+        ovnDBCluster:
+          ovndbcluster-nb:
+            dbType: NB
+            storageRequest: 10G
+            networkAttachment: internalapi
+          ovndbcluster-sb:
+            dbType: SB
+            storageRequest: 10G
+            networkAttachment: internalapi
+        ovnNorthd:
+          networkAttachment: internalapi
+          replicas: 1
```

Comment:
OVN northd is supposed to start after DB imports are done, but it's already not the case ( https://github.com/openstack-k8s-operators/data-plane-adoption/issues/109 ) so not a blocker. Eventually we should probably have replicas 0 here and replicas 1 in the OVN adoption.

However, let's please make the same updates to docs as we're doing to the test suite.

Reply:
Since the CI is green here the doc update can be a different PR but it would be nice to have it posted before we merge?

Reply:
I'm not sure what exactly should be in that follow-up, sorry

Reply:
Sorry i see you're editing both, not sure how i missed that earlier, i guess e_too_many_things :)